torch/fx/experimental/Partitioner.py

```diff
@@ -1,7 +1,92 @@
 from torch.fx.graph_module import GraphModule
 from torch.fx.node import Node
-from torch.fx.experimental import GraphManipulation
-from typing import Dict, List, Union
+from typing import Dict, List, Union, Tuple
+from torch.fx.experimental.GraphManipulation import get_all_users_of
+def _get_users_for_all_nodes(fx_module: GraphModule) -> Dict[int, List[int]]:
```

Comment:
This will not be needed after https://github.com/pytorch/pytorch/pull/45775

Reply:
I have changed the return to Dict[Node, List[Node]]. The purpose for this is for _get_args_for_all_nodes(). After we get each node and its user nodes, we could get each node and its arg nodes in one pass.